### PR TITLE
APPS-1517 APPS-1568 terminate errorStrategy - failed subjob should fail

### PR DESCRIPTION
**Pull request by mhrvol**

### CODE REVIEW

**Comment by mhrvol** [comment]
Records to be added in all regions.

File: src/python/dxpy/templating/templates/nextflow/src/nextflow.sh [hunk 1]
```diff
@@ -406,6 +406,14 @@ nf_task_exit() {
   # mark the job as successful in any case, real task
   # error code is managed by nextflow via .exitcode file
   if [ -z ${exit_code} ]; then export exit_code=0; fi
+
+ terminate_record=$(dx find data --name $DX_JOB_ID --path $DX_WORKSPACE_ID --brief | head -n 1)
```
Comment (commandlinegirl) [inline src/python/dxpy/templating/templates/nextflow/src/nextflow.sh:410]: Could you fix the indent?

Comment (commandlinegirl) [reply src/python/dxpy/templating/templates/nextflow/src/nextflow.sh:410]: Also, if you could add the comment why we need this?

**Review by commandlinegirl — CHANGES_REQUESTED** [review]

File: src/python/dxpy/templating/templates/nextflow/src/nextflow.sh (hunk 1 above)
Comment (mhrvol) [reply src/python/dxpy/templating/templates/nextflow/src/nextflow.sh]: Fixed and commented. Also added few echo messages for the user.

**Review by commandlinegirl — APPROVED** [review]
LGTM, thank you @mhrvol

**Pull request merged**